linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/conf/EntranceConfiguration.scala

```diff
@@ -225,4 +220,7 @@ object EntranceConfiguration {
   val CREATOR_IP_SWITCH =
     CommonVars("wds.linkis.entrance.user.creator.ip.interceptor.switch", false)
 
+  val SEND_INSTANCE =
```

Comment:
rename to ENABLE_ENTRANCE_DIRTY_DATA_CLEAR  and linkis.entrance.auto.clean.dirty.data.enable

Reply:
done